src/Symfony/Bundle/WebProfilerBundle/DependencyInjection/Configuration.php

```diff
@@ -46,6 +46,7 @@ public function getConfigTreeBuilder()
                 ->end()
                 ->booleanNode('intercept_redirects')->defaultFalse()->end()
                 ->scalarNode('excluded_ajax_paths')->defaultValue('^/(app(_[\\w]+)?\\.php/)?_wdt')->end()
+                ->booleanNode('help')->defaultFalse()->end()
```

Comment:
As Symfony already has lots of config options, it's often tried to avoid adding many more options. I think this is a case where a config option is not required. Always enabling help will only add some minor question mark icons, which aren't that distracting (and maybe even helpfull for everyone). Always enabling it will also make code a lot easier (avoiding many if statements).